apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -29,18 +33,31 @@ export const StatusBar = ({
     success: "success",
     error: "error",
   };
+
+  const resetMutation = trpc["app.reset"].useMutation();
+  const [showRestartModal, setShowRestartModal] = useState(false);
+
+  const restart = useCallback(async () => {
+    setShowRestartModal(false);
+    await resetMutation.mutateAsync();
+  }, [resetMutation]);
+
   return (
     <footer
       className={classNames(
-        theme.bg3,
+        theme.bg1,
         theme.text1,
         theme.border3,
-        "py-1 px-4 flex text-2xs w-full relative z-10",
+        "px-4 flex items-center text-2xs w-full relative z-10",
```

Comment:
```suggestion
        "px-4 flex gap-2 items-center text-2xs w-full relative z-10",
```

Standardize gap. This gap would apply if the width of the window is small.